deploy/gke/simple/README.md

```diff
@@ -67,6 +67,171 @@ Since there is no application currently running on the cluster, the collector is
+
+**NOTE: You will need to update the timestamps in [testdata-metrics](./otlp-data/testdata-metrics.json) to be within 24 hours of current time otherwise the metrics will not show up in Google Cloud console's Metrics Explorer. You might also need to update timestamps in other testdata - metrics and logs in case they become old enough to not be recognized by Google Cloud.**
+
+### Seeing the telemetry data
+
+At this point, we have several files acting as sources of telemetry data and a reciever configured withing the OpenTelemetry Collector that listens to these files whenever there are updates made to them. The collector also receives telemetry data from the files once when they are initially addedd (mounted).
+
+So there should already be telemetry data emitted which should have been caught by the Collector. The current collector configuration has 2 exporters configured to where these traces will be exported -  
+1. `googlecloud`
+     - You can log into your Google Cloud console and then use [Trace List](https://console.cloud.google.com/traces/list) to look for traces, [Metrics Explorer](https://console.cloud.google.com/monitoring/metrics-explorer) or [Cloud Logging](https://console.cloud.google.com/logs) for logs.
+ 2. `logging` 
+     - You can view the telemetry data on stdout for the pod container, run `kubectl logs <pod_container_name> -n $OTEL_NAMESPACE`. 
+
+### Making a change to the telemetry data (Optional)
+
+You might want to update the telemetry data being recieved by the collector to add new traces, metrics or logs or update any other attributes. To do this you need to make a change to the ConfigMap that is created from our test data file. To do this, you need to -
+1. Update the telemetry data in the desired data file - [testdata-metrics.json](./otlp-data/testdata-metrics.json), [testdata-traces.json](./otlp-data/testdata-traces.json). Make sure that JSON here is **not** pretty printed and remains minified. 
+2. Update the corresponding ConfigMap. For instance, if you made changes to `testdata-traces.json` file, you need to update `otlp-test-data-traces` - 
+    ```
+    kubectl create configmap otlp-test-data-traces --from-file=./otlp-data/testdata-traces.json -n $OTEL_NAMESPACE --dry-run=client -o yaml | kubectl apply -f -
+    ```
+
+**NOTE**: It may take some time *(usually a few seconds)* for the config file to be updated.
+
+To check if ConfigMap is updated, use following command with desired ConfigMap name - 
+```
+ kubectl describe configmaps otlp-test-data-traces -n $OTEL_NAMESPACE 
+```
+
+To check if the test file at mount path is updated, use following command with correct file path - 
+```
+ kubectl exec -n $OTEL_NAMESPACE -it  <pod_container_name> -- cat /mnt/testdata/traces/testdata-traces.json 
```

Comment:
this will only work if they're following the troubleshooting doc right? Maybe it should be moved there

Reply:
Yes, it does seem similar to the toubleshooting step but this is different from troubleshooting. 

This is in case the user wants to update their JSON data. Like it is possible that user reaches this step without needing to open troubleshooting, but I can remove this and link the troubleshooting section since content is same.

Reply:
Yeah, the problem is that this command won't work unless they do the steps to update the Dockerfile and give them a shell in the container

Reply:
Oh yes, I see your point. I agree, better to move it there.